src/constants.js

```diff
@@ -81,30 +79,30 @@ export const AGE_KEYS = {
   over40: "40<",
 };
 
-export const AGES = {
-  [AGE_KEYS.under25]: "<25",
-  [AGE_KEYS["25_29"]]: "25-29",
-  [AGE_KEYS["30_34"]]: "30-34",
-  [AGE_KEYS["35_39"]]: "35-39",
-  [AGE_KEYS.over40]: "40<",
+const AGES = new Map([
```

Comment:
I made all of these demographic collections `Maps` to preserve insertion order similar to the `DIMENSION_MAPPINGS`